### PR TITLE
Add draft for a page about the wallet software

### PR DESCRIPTION
**Pull request by kevinsimper**
When I first discovered Ethereum, I read about it on the main homepage and not on ethdocs.

I think there should be a page that explains what the wallet software is and does. You want to get a quick introduction to what it is and not just download it straight away as that is a really slow process if you just want to learn and read about Ethereum.

You should also be told that the Ethereum Wallet is not the only way to use Ethereum as many newcomers think. There is a lot of other really cool projects in the community that enables people to interact with the Ethereum blockchain 😄 

But because of that you should be told of the awesome benefits you get by running a full node.

As many people are reading and browsing on smartphones, recommendations should really be targets towards mobile.

image

### CODE REVIEW

**Comment by alexvandesande** [comment]
Thanks Kevin. This is actually a very good idea. I'll review the text

**Comment by kevinsimper** [comment]
@alexvandesande Do you have something that could make this better? :)

**Comment by kevinsimper** [comment]
@alexvandesande Something I can do to move this forward? 😄

File: views/content/wallet.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Ethereum Wallet
+
+The Ethereum Wallet app that you can download allows you to run the full block chain on your own computer. It allows you very easy to be part of the blockchain.
```
Comment (alexvandesande) [inline views/content/wallet.md]: The Ethereum Wallet is an app that allows you to interact with the Ethereum blockchain on your own computer easily.

File: views/content/wallet.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+![](/images/wallet-screenshot.png)
+
+With the wallet app you can create new wallets and transfer ether really easily. You can also interact with smart contracts that other have deployed to the Ethereum blockchain.
```
Comment (alexvandesande) [inline views/content/wallet.md]: Ethereum Wallet allows you to receive and transfer ether or any ethereum compatible token, as well as allowing you to deploy a contract or interact with any smart contract deployed on the network. The Wallet supports the Main ethereum chain, test chains and even private networks.

File: views/content/wallet.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+You don't absolute need the wallet to hold ether. There is different places that also allow you to interact with the Ethereum blockchain such as projects like
```
Comment (alexvandesande) [inline views/content/wallet.md]: I feel that pointing to other projects should be elsewhere in the home page, let's remove it from here.

File: views/content/wallet.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+- MyEthereumWallet
+- Metamask
+
+Running the wallet app and thereby having the whole blockchain on your computer is called running a "full node". This is crucial to the success of blockchains because your are participating in the network that ensures that there is a single truth.
```
Comment (alexvandesande) [inline views/content/wallet.md]: We intend to allow the option of light client nodes too, so I wouldnt tie this page to running a full node.

File: views/wallet.jade [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+      div.row.scrollme                
+        div.col-md-12
+          include:md content/wallet.md
+          a(href="/" style="display: block;").button.ethereum-wallet Download the wallet from the frontpage
+          a.button.small(href='https://github.com/ethereum/ethereum-org/edit/master/views/content/token.md') Improve this page
+
+  
+  script
+    |  ((window.gitter = {}).chat = {}).options = {
+    |    room: 'ethereum/tutorials'
```
Comment (alexvandesande) [inline views/wallet.jade]: change to ethereum/mist

File: views/content/wallet.md (hunk 4 above)
Comment (kevinsimper) [inline views/content/wallet.md]: Yes, but I feel like it makes sense to talk about since that is what you are doing?

**Comment by kevinsimper** [comment]
@alexvandesande I updated it to your review 😄

**Comment by kevinsimper** [comment]
@alexvandesande Is there anything more I should do? 👍

**Comment by kevinsimper** [comment]
@alexvandesande Hi Alex, can I help with something?

@cubedro can you help? :)

**Pull request merged**